### ISSUE TITLE
error during install strechr/testify, undefined: errors.Is

### ISSUE THREAD

**Issue by raghuiisc** [stretchr/testify#1091]
# github.com/stretchr/testify/assert
/root/go/pkg/mod/github.com/stretchr/testify@v1.7.0/assert/assertions.go:1704:5: undefined: errors.Is
/root/go/pkg/mod/github.com/stretchr/testify@v1.7.0/assert/assertions.go:1727:6: undefined: errors.Is
/root/go/pkg/mod/github.com/stretchr/testify@v1.7.0/assert/assertions.go:1750:5: undefined: errors.As
/root/go/pkg/mod/github.com/stretchr/testify@v1.7.0/assert/assertions.go:1767:7: undefined: errors.Unwrap
/root/go/pkg/mod/github.com/stretchr/testify@v1.7.0/assert/assertions.go:1771:7: undefined: errors.Unwrap
note: module requires Go 1.13

I am getting this error after i upgraded to "github.com/go-redis/redis/v8 v8.10.0"

**Comment by brackendawson** [comment]
It looks like you are using a go version lower than 1.13, is that right? Currently testify requires [at least 1.13](https://github.com/stretchr/testify#supported-go-versions). Please run `go env` and paste the result.

**Comment by raghuiisc** [comment]
Thanks once i updated the go version it worked fine.

**Comment by NamrataPatel29** [comment]
I updated my go version to 1.13 but my circle ci tests are still failing with the following error 
=== Errors
/go/pkg/mod/github.com/stretchr/testify@v1.7.0/assert/assertions.go:1704:5: undefined: errors.Is
/go/pkg/mod/github.com/stretchr/testify@v1.7.0/assert/assertions.go:1727:6: undefined: errors.Is
/go/pkg/mod/github.com/stretchr/testify@v1.7.0/assert/assertions.go:1750:5: undefined: errors.As
/go/pkg/mod/github.com/stretchr/testify@v1.7.0/assert/assertions.go:1767:7: undefined: errors.Unwrap
/go/pkg/mod/github.com/stretchr/testify@v1.7.0/assert/assertions.go:1771:7: undefined: errors.Unwrap
note: module requires Go 1.13

can anyone help me with this?

**Comment by brackendawson** [comment]
@NamrataPatel29 your Circle CI is definitely not running go 1.13: https://golang.org/doc/go1.13#errors

**Comment by NamrataPatel29** [comment]
@brackendawson yes it was not running go 1.13 , the issue got resolved when it ran go 1.13.

Also i am facing a similar issue whem i am tried updating another dependency (github.com/aws/aws-sdk-go from 1.30.23 to 1.40.10)

=== Errors
/go/pkg/mod/golang.org/x/net@v0.0.0-20210614182718-04defd469f4e/http2/client_conn_pool.go:305:6: undefined: errors.Is
note: module requires Go 1.17

does this require updation of go to 1.17 ? Isnt error.Is part of Go 1.13 ?

**Comment by colin-valentini** [comment]
This issue should be closed.
